### DISCUSSION TITLE
Welcome to SEHeadTracking Discussions!

### DISCUSSION THREAD

**Discussion by jarno-r** [Corben-SpacedOut/SEHeadTracking#5] [Announcements]
## 👋 Welcome!
Any suggestions, questions or ideas for the Head Tracking plugin can be left here.

**Comment by qwuille** [comment]
Really nice! Love it, one thing i wonder tho, is there a possibility when controlling turrets to deactivate the headtracking?

**Reply by jarno-r** [reply]
Good suggestion! It should really be completely disabled when controlling turrets. I made a reminder to fix the issue: https://github.com/Corben-SpacedOut/SEHeadTracking/issues/6. It might take a while though, since I haven't been actively working on this for 4 years.
